pkgs/applications/misc/lttoolbox/default.nix

```diff
@@ -0,0 +1,37 @@
+{ fetchFromGitHub, lib, pkgs, stdenv }:
+
+stdenv.mkDerivation {
+  src = fetchFromGitHub {
+    owner = "apertium";
+    repo = "lttoolbox";
+    rev = "v3.7.1";
+    sha256 = "sha256-3lHXKtwQSrMGQEGOGr27e3kB2qKkTFZcEzeAnIm89Rg=";
+  };
+  name = "lttoolbox";
+  buildInputs = [
+    pkgs.autoconf
+    pkgs.automake
+    pkgs.libtool
+    pkgs.libxml2
+    pkgs.libxml2.dev
+    pkgs.icu
+    pkgs.pkg-config
+    pkgs.utf8cpp
+  ];
+  buildFlags = [
+    "CPPFLAGS=-I${pkgs.utf8cpp}/include/utf8cpp"
+  ];
+  configurePhase = ''
+    ./autogen.sh --prefix $out
+  '';
+  doCheck = true;
+  checkPhase = ''
+    ${pkgs.python3}/bin/python3 tests/run_tests.py
+  '';
+
+  meta = with lib; {
```

Comment:
You should add license information here (in this case, [GPL v2](https://github.com/apertium/lttoolbox/blob/main/COPYING)).